AI/StupidAI/StupidAI.cpp

```diff
@@ -310,9 +310,9 @@ BattleAction CStupidAI::goTowards(const BattleID & battleID, const CStack * stac
 		}
 	}
 
-	BattleHex bestNeighbor = hexes.front();
+	BattleHex bestneighbour = hexes.front();
```

Comment:
Why was this one renamed?
```suggestion
	BattleHex bestNeighbour = hexes.front();
```